moveit_ros/moveit_servo/test/pose_tracking_test.cpp

```diff
@@ -155,12 +199,10 @@ TEST_F(PoseTrackingFixture, OutgoingMsgTest)
 
 int main(int argc, char** argv)
 {
-  ros::init(argc, argv, LOGNAME);
-  testing::InitGoogleTest(&argc, argv);
-
-  ros::AsyncSpinner spinner(8);
-  spinner.start();
+  ::testing::InitGoogleTest(&argc, argv);
+  rclcpp::init(argc, argv);
```

Comment:
Should the ros init be called before the google test init?

Reply:
I don't think so (ROS1 example - [http://wiki.ros.org/gtest])